pyke/meta.yaml

```diff
@@ -11,7 +11,8 @@ source:
     sha1: 6f6bfab243564c17218d81fa40bc3bf24e0d5847  # [py3k]
 
 build:
-    number: 1
+    number: 1  # [not osx]
+    number: 2  # [osx]
```

Comment:
I can see pyke for OSX and Python3 [here](http://anaconda.org/SciTools/pyke/files)  why the build bump?  Also, splitting the building number adds an extra maintenance burden that I don't believe it is worth the benefits.